examples/TestEm3/electrons.cu

```diff
@@ -24,15 +24,22 @@
 #include <G4HepEmElectronInteractionIoni.icc>
 #include <G4HepEmElectronInteractionUMSC.icc>
 #include <G4HepEmPositronInteractionAnnihilation.icc>
+#ifndef NOFLUCTUATION
+#include <G4HepEmElectronEnergyLossFluctuation.icc>
+#endif
+
+__device__ char g_nextInteractionForEl[8'000'000];
+__device__ char g_nextInteractionForPos[8'000'000];
```

Comment:
Could you quickly elaborate to me please, why we cannot add this information to the `Track` itself? Then we would not need this additional global state and the checking of whether you have allocated enough of it.